data/data_test.go

```diff
@@ -918,7 +910,6 @@ func (s *ManagerTestSuite) TestGetConfig(t *C) {
 		Encoding:     "none",
 		SendInterval: 1,
 	}
-	bytes, _ := json.Marshal(config)
```

Comment:
Comparing json as string is prone to constant errors as properties in json are not ordered in anyway so turning json into a string might lead to have different string every time e.g.
```
{
    "id": 1,
    "name": "A green door"
}
```
is the same json as
```
{
    "name": "A green door",
    "id": 1
}
```
but it's not the same string.
So in tests I switched from comparing jsons as string to comparing Unmarshaled structs which fixed the issues.